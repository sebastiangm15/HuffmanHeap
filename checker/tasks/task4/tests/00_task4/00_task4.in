5
19 SIK27
8 SBH1O
3 SO0ZT
16 S4UBM
17 SKVS1
2
S4UBM
SBH1O
