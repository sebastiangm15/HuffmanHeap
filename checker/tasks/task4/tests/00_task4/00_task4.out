SO0ZTSBH1OS4UBM
